spec/controllers/job_definitions_controller_spec.rb

```diff
@@ -38,6 +38,67 @@
 
       expect(assigns(:definition)).to be_new_record
     end
+
+    context 'with dup_from params' do
+      subject! { get :new, params: { dup_from: definition.id } }
+      it do
+        expect(response).to have_http_status(:ok)
+        expect(response).to render_template('new')
+
+        expect(assigns(:definition)).to be_new_record
+        expect(assigns(:definition).name).to eq definition.name
+        definition.admins << controller.current_user unless definition.admins.include?(controller.current_user)
```

Comment:
It seems to better for do not change `definition` instance in `it` block.
How about changing it to the below code?

```ruby
expect(assigns(:definition).admins).to eq(definition.admins + controller.current_user)
```